editor/editor_inspector.cpp

```diff
@@ -918,6 +918,9 @@ void EditorProperty::gui_input(const Ref<InputEvent> &p_event) {
 
 	if (me.is_valid()) {
 		Vector2 mpos = me->get_position();
+		if (bottom_child_rect.has_point(mpos)) {
+			return; // Makes child EditorProperties behave like sibling nodes.
```

Comment:
How does that work exactly?

Reply:
Just when handling mouse events.  

In this case, when clicking on the `Label` of a child `EditorProperty`, we don't want the parent `EditorProperty`'s menu to pop up (RMB), but we want the **root** inspector to have focus (LMB).